src/r2mm/installing/default_installation_rules/game_rules/InstallRules_GDWeave.ts

```diff
@@ -0,0 +1,32 @@
+import type { CoreRuleType } from '../../InstallationRules';
+import * as path from 'path';
+import { GAME_NAME } from '../../profile_installers/ModLoaderVariantRecord';
+import { RuleSubtype } from "../../InstallationRules";
+
+export function buildGDWeaveRules(gameName: GAME_NAME, extraRules?: RuleSubtype[]): CoreRuleType {
```

Comment:
This whole definitions shouldn't be needed, as GDWeaveInstaller.ts provides custom installation logic for the mod loader (and mods). Note the ReturnOfModding example though: the PackageInstaller implementations can call IntallRules internally if it makes implementation easier. In GDWeave's case the process is so straightforward though, that it might not be needed.